server/data/testData/thrownErrors.ts

```diff
@@ -8,8 +8,8 @@ import { SanitisedError } from '../../sanitisedError'
  */
 // eslint-disable-next-line import/prefer-default-export
 export function mockThrownError<T>(responseBody: T, status: number = 400): SanitisedError<T> {
-  const error = new SanitisedError<T>(`Error: ${status}`)
```

Comment:
`SanitisedError` is a real class extending error so sure we'd want to instantiate the real thing still?

Reply:
That's what I tried at first, but I'm getting a bunch of `TypeError: hmpps_rest_client_1.SanitisedError is not a constructor`

Reply:
i wonder if those errors happened before you swapped in the `SanitisedError` from the lib? cos the one in the app before was not a constructor… but surely this one is cos it's just a class… :/

Reply:
Maybe it's because [it's exported as a `Type`](https://github.com/ministryofjustice/hmpps-typescript-lib/blob/0ae5651e0d8e0b44e6eeacfc3b108fc9fb1375be/packages/rest-client/src/main/index.ts#L10) rather than as a class?

Reply:
that's likely the problem. also, i take it back cos there's no real class instantiated here anyway:
https://github.com/ministryofjustice/hmpps-typescript-lib/blob/6fa730c39882f4b6411fc81f2bea38b2be69a6e8/packages/rest-client/src/main/helpers/sanitiseError.ts#L8